packages/transport-webrtc/src/index.ts

```diff
@@ -280,6 +280,7 @@ export interface TransportCertificate {
    * The hash of the certificate
    */
   certhash: string
+  notAfter: string
```

Comment:
This would be better as a `number` so we don't have to convert the string back to a date in order to see if the cert is about to expire, instead it can just be compared to `Date.now() - threshold `.